src/api/app/components/chart_component.html.haml

```diff
@@ -1,10 +1,12 @@
-- raw_data = build_results_data
 - if raw_data.size.positive?
   .m-4
+    .btn.btn-outline-primary.float-end{ onclick: 'updateChartBuildResults()', accesskey: 'r', title: 'Refresh Chart Build Results' }
+      Refresh
+      %i.fas.fa-sync-alt{ id: 'chart-build-reload' }
     %h4
       Build results summary
-    - if distinct_repositories(raw_data).size > 5 || raw_data.size > 12
-      = column_chart( chart_data(raw_data),
+    - if distinct_repositories.size > 5 || raw_data.size > 12
```

Comment:
Can we transform those numbers into constants with a meaningful name?

Reply:
Magic numbers are a code smell https://stackoverflow.com/a/47902

Reply:
done :+1: